### ISSUE TITLE
When screen reader running, and folder opened in vsCode explorer, window becomes unnavicable

### ISSUE THREAD

**Issue by RichCaloggero** [microsoft/vscode#292069]
Type: <b>Bug</b>

1. with screen reader running (using NVDA on windows 11), start vsCode (release and insider have same issue)
2. add a folder to the workspace via vsCode explorer or via the "add folder to workspace" command
3. NVDA can now no longer see anything in the window; pressing tab or other keys causes the screen reader to announce "unknown". You must unfocus vsCode, restart screen reader, then refocus vsCode.

Because I cannot see, I'm not sure what is happening on screen, but I was able to start narrator and see a prompt to enable trust in the new folder / workspace. However, this tends to disappear randomly so I cannot catch it consistently.

VS Code version: Code - Insiders 1.109.0-insider (ea084045c1ea954f98f53a5a7c31a85a1877b100, 2026-01-30T10:24:26.004Z)
OS version: Windows_NT x64 10.0.26100
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) Ultra 7 265 (20 x 2381)|
|GPU Status|2d_canvas: enabled<br>GPU0: VENDOR= 0x8086, DEVICE=0x7d67 [Intel(R) Graphics], DRIVER_VENDOR=Intel, DRIVER_VERSION=32.0.101.8132 *ACTIVE*<br>GPU1: VENDOR= 0x1414, DEVICE=0x008c [Microsoft Basic Render Driver], DRIVER_VERSION=10.0.26100.7309<br>Machine model name: <br>Machine model version: <br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>trees_in_viz: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.46GB (22.92GB free)|
|Process Argv|--crash-reporter-id fff8e396-d226-4327-acbe-209733d94f8d|
|Screen Reader|yes|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
pythonvspyt551:31249597
nativeloc1:31118317
dwcopilot:31158714
dwoutputs:31242946
copilot_t_ci:31333650
g012b348:31231168
pythonrdcb7:31268811
pythonpcpt1cf:31399617
6518g693:31302842
4f60g487:31327383
6abeh943:31336334
envsactivate1:31349248
editstats-enabled:31346256
cloudbuttont:31366566
3efgi100_wstrepl:31403338
use-responses-api:31390341
none:31438490
cp_cls_c_966_ss:31454199
chatvisible:31417128
4je02754:31426859
a5gib710:31434435
7ig2g208:31429686
anthropic_thinking_t:31427198
406hc587_ask_agent:31428392
cp_cls_c_1081:31454833
ia-use-proxy-models-svc:31446143
a43f0574a:31442822
e9c30283:31453065
control_6dc23131:31449957
idci7584:31454084
edit_mode_hidden:31453871
e3e4d672:31454087
e7aae401:31452367
i54ji102:31455230
55364912:31455227
7ef87755:31455235

```

</details>

**Comment by meganrogge** [comment]
Testing in oss off of main, I am asked if I trust the folder, which seems correct. Thus closing.
